src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -267,6 +266,26 @@ private function getResourceFieldConfiguration(string $resourceClass, ResourceMe
         return null;
     }
 
+    private function getGraphQlFilterType(array $value, int $depth)
+    {
+        $nullable = isset($value['required']) ? !$value['required'] : true;
+
+        if ('enum' === $value['type']) {
+            if (!isset($this->graphqlTypes['#OrderEnumType'])) {
+                $this->graphqlTypes['#OrderEnumType'] = new EnumType([
+                    'name' => 'OrderEnumType',
```

Comment:
This is not the most scalable solution. I would be glad for any suggestions how to improve this.